app/views/teacher_training_adviser/steps/_has_teacher_id.html.erb

```diff
@@ -1,6 +1,16 @@
-<% @page_title = "Do you have your previous teacher reference number?" %>
-
-<%= f.govuk_radio_buttons_fieldset(:has_id, inline: true, legend: { tag: "h1" }) do %>
+<%= f.govuk_radio_buttons_fieldset(:has_id, inline: true, legend: { tag: "h1", text: "Do you have a teacher reference number (TRN)?" }) do %>
   <%= f.govuk_radio_button :has_id, true, label: { text: 'Yes' }, link_errors: true %>
   <%= f.govuk_radio_button :has_id, false, label: { text: 'No' }, link_errors: true %>
 <% end %>
+
+<details class="govuk-details" data-module="govuk-details">
+  <summary class="govuk-details__summary">
+    <span class="govuk-details__summary-text">
+      What is a teacher reference number (TRN)?
+    </span>
+  </summary>
+  <div class="govuk-details__text">
+    A TRN is a 7 digit number that you get if you hold qualified teacher status in England or Wales or you contribute to the Teachers' Pension Scheme.<br>
+    If you're eligible for a TRN in England, you should be allocated one automatically.
```

Comment:
Please add another line break here. It doesn't appear consistent.

Reply:
delete this line